container-accesslogging/src/main/java/com/yahoo/container/logging/LogFileHandler.java

```diff
@@ -263,15 +264,28 @@ private void internalRotateNow() throws InterruptedException {
         numberOfRecords = 0;
         lastRotationTime = now;
         nextRotationTime = 0; //figure it out later (lazy evaluation)
-        if (compressOnRotation && (oldFileName != null)) {
-            triggerCompression(oldFileName);
+        if ((oldFileName != null)) {
+            if (compressOnRotation) {
+                triggerCompression(oldFileName);
+            } else {
+                NativeIO nativeIO = new NativeIO();
+                nativeIO.dropFileFromCache(new File(oldFileName));
+            }
         }
     }
 
     private void triggerCompression(String oldFileName) {
         try {
+            String zippedFileName = oldFileName + ".gz";
```

Comment:
nitpick: zipped implies .zip ending, maybe call it gzipped

Reply:
Fixed